### ISSUE TITLE
Consider a bower package?

### ISSUE THREAD

**Issue by fraserxu** [floydwch/angular-underscore#9]
I've been using this in my project, but we are using `bower` as the package manager. Have you consider to make it installable through bower?

BTW, in order to use this, I also have to include `underscore` in my `bower.json` file, I want something like `angular-underscore` in my json file, thanks!

**Comment by floydwch** [comment]
I've added `bower.json` without testing, would you mind to test it?

And since considering `Lo-Dash`, the decision of `underscore` or `Lo-Dash` as the infrastructure is remained with users, i.e., you can include `'underscore'` or `'lodash'`, depends on you .

**Comment by fraserxu** [comment]
If you say so, why not have another one named `angular-lodash`. The name would be confusing if you let people choose which library to use, no?

**Comment by floydwch** [comment]
@fraserxu Because `Lo-Dash` is as a drop-in replacement for `underscore`, if include `underscore` directly in this package, one can't have other choice for consideration, so leave it for users might be better.

Also, this package's description is "Underscore adapter for AngularJS", i.e., it's just an adapter, not an implementation of `underscore`.

**Comment by sentenza** [comment]
@floydsoft I've tested installing ver. 0.4.0 but I have this error in return:

```
bower angular-underscore#~0.4.0     ENORESTARGET No tag found that was able to satisfy ~0.4.0
```

**Comment by Zmetser** [comment]
Indeed. Tagging the releases would be nice. But in the meantime just do `bower angular-underscore` it'll check out the master.

`bower info angular-underscore`

**Comment by sentenza** [comment]
Yes, I already did that to fix the dependency.

**Comment by fraserxu** [comment]
@sentenza You can only try install it directly without adding the version.

**Comment by tomaszkrym** [comment]
+1 for tagging the version - my webapp stopped working because of the fact, that angular-underscore was updated to latest version, while I had older version of underscore in bower.

**Comment by floydwch** [comment]
@tomaszkrym I added the tag v0.5.0, could you mind to test for it?

**Comment by tomaszkrym** [comment]
@floydsoft it works fine, thanks!
